pkg/scheduler/framework/preemption/preemption.go

```diff
@@ -182,7 +182,7 @@ func (ev *Evaluator) Preempt(ctx context.Context, pod *v1.Pod, m framework.NodeT
 	// 4) Find the best candidate.
 	bestCandidate := ev.SelectCandidate(candidates)
 	if bestCandidate == nil || len(bestCandidate.Name()) == 0 {
-		return nil, framework.NewStatus(framework.Unschedulable)
+		return nil, framework.NewStatus(framework.Unschedulable, "no candidates in the preemption")
```

Comment:
what's a candidate? a pod or a node?

Better say `for preemption` instead of `in the preemption`.

Reply:
done